### PR TITLE
Updated deprecated test code

### PR DESCRIPTION
**Pull request by immetoo**

### CODE REVIEW

File: web-tck/src/main/java/ws/ament/hammock/web/tck/ServletTest.java [hunk 1]
```diff
@@ -40,7 +39,7 @@
 @RunWith(Arquillian.class)
 public abstract class ServletTest {
     public static JavaArchive createArchive(Class<?>... classes) {
-        String property = System.getProperty(Bootstrapper.class.getName(), Weld3Bootstrapper.class.getName());
+        String property = System.getProperty(Bootstrapper.class.getName(), "ws.ament.hammock.bootstrap.weld3.Weld3Bootstrapper");
```
Comment (johnament) [inline web-tck/src/main/java/ws/ament/hammock/web/tck/ServletTest.java]: Now that I'm seeing this, this property can be removed.  Can you try without it?

File: web-tck/src/main/java/ws/ament/hammock/web/tck/FilterTest.java [hunk 2]
```diff
@@ -30,34 +29,40 @@
...
-
     @Test
-    public void shouldBootWebServerWithOnlyFilter() throws Exception {
-        System.out.println("open http://localhost:8080/");
-//        Thread.sleep(20000);
-        try (InputStream stream = new URL("http://localhost:8080/").openStream()) {
-            String data = IOUtils.toString(stream).trim();
-            assertThat(data).isEqualTo("Hello, world!");
-        }
+    public void shouldBootAndOnlyFilter() throws Exception {
```
Comment (johnament) [inline web-tck/src/main/java/ws/ament/hammock/web/tck/FilterTest.java]: Please leave the method named as `shouldBootWebServerWithOnlyFilter` - this was to fix a specific issue with tomcat integration, where if you only had filters defined (via SparkJava) the app didn't deploy.

File: web-tck/src/main/java/ws/ament/hammock/web/tck/FilterTest.java [hunk 3]
```diff
@@ -30,34 +29,40 @@
...
+        .when()
+            .get("/");
+    }
 
-        try (InputStream stream = new URL("https://localhost:8443/rest").openStream()) {
-            String data = IOUtils.toString(stream).trim();
-            assertThat(data).isEqualTo("Hello, world!");
-        }
+    @Test
+    public void shouldBootAndOnlyFilterSecure() throws Exception {
```
Comment (johnament) [inline web-tck/src/main/java/ws/ament/hammock/web/tck/FilterTest.java]: Same, I would recommend the name `shouldBootWebServerWithOnlyFilterViaHttps` or `viaSecure`

File: web-tck/src/main/java/ws/ament/hammock/web/tck/ServletTest.java (hunk 1 above)
Comment (immetoo) [reply web-tck/src/main/java/ws/ament/hammock/web/tck/ServletTest.java]: Yes works with mvn test.
Only when running test by ide it fails.

File: web-tck/src/main/java/ws/ament/hammock/web/tck/FilterTest.java (hunk 2 above)
Comment (immetoo) [reply web-tck/src/main/java/ws/ament/hammock/web/tck/FilterTest.java]: have renamed both methods

**Pull request merged**